gpu/octree/src/octree.cpp

```diff
@@ -66,6 +66,11 @@ pcl::gpu::Octree::Octree() : cloud_(nullptr), impl(nullptr)
     int bin, ptx;
     OctreeImpl::get_gpu_arch_compiled_for(bin, ptx);
 
+    if (bin < 0 || ptx < 0)
+    {
+        pcl::gpu::error("cudaFuncGetAttributes() returned a value < 0.\nThis is likely a build configuration error.\nEnsure that the proper compute capability is specified in the CUDA_ARCH_BIN cmake variable when building for your GPU.", __FILE__, __LINE__);
```

Comment:
Maybe use raw string literal to showoff those neat newlines

Reply:
Sure. Why not :).